### PR TITLE
Port disk integration test for Windows

### PR DESCRIPTION
**Pull request by gut**

### CODE REVIEW

File: daisy/compute/compute.go [hunk 1]
```diff
@@ -289,6 +291,16 @@ func (c *client) AttachDisk(project, zone, instance string, d *compute.AttachedD
 	return c.i.zoneOperationsWait(project, zone, op.Name)
 }
 
+// DetachDisk detaches a GCE persistent disk to an instance.
+func (c *client) DetachDisk(project, zone, instance string, d *compute.Disk) error {
```
Comment (adjackura) [inline daisy/compute/compute.go]: Just take the name of the disk to detach, no need for the full Disk resource

File: daisy/compute/compute.go [hunk 2]
```diff
@@ -848,6 +860,17 @@ func (c *client) InstanceStopped(project, zone, name string) (bool, error) {
 	}
 }
 
+// ResizeDisk resizes a GCE persistent disk. You can only increase the size of the disk.
+func (c *client) ResizeDisk(project, zone string, d *compute.Disk) error {
```
Comment (adjackura) [inline daisy/compute/compute.go]: Take name of disk and compute.DisksResizeRequest

func (c *client) ResizeDisk(project, zone, disk string, drr *compute.DisksResizeRequest) error {

**Comment by gut** [comment]
> Can you separate the daisy step additions from the image test updates
I did it. They are different commits. Do you separating PRs?

**Comment by adjackura** [comment]
Yeah, have them as separate PR's, we squash commits on merge

**Comment by gut** [comment]
alright. I'll then leave this one for image test updates and it should only be integrated after the daisy step additions are on master.

**Comment by gut** [comment]
Depends on #476

**Pull request merged**